src/firebase/models/rooms.js

```diff
@@ -0,0 +1,43 @@
+import {
+  collection,
+  doc,
+  onSnapshot,
+  setDoc,
+  updateDoc,
+  getDoc
+} from "firebase/firestore";
```

Comment:
In this file, we're managing rooms, and this imports the core functions we need to interact with firestore to do this